tests/test_data/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
Is this just a file moved from a different directory? 

Is it common for this sort of thing to go in an `__init__.py` file? It seems like that's usually just reserved for aliasing and imports. One alternative would be to have something called like `test_setup.py` or something in this directory and have `__init__.py` pull it in.